code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -217,16 +219,6 @@
 	var/mob/living/carbon/human/interactive/greytide/clone = new(src)
 	clone.hardset_dna(H.dna.uni_identity,H.dna.struc_enzymes,H.real_name, H.dna.blood_type, H.dna.species.type, H.dna.features)
 
-	//There's no define for this / get all items ?
-	var/list/slots = list(slot_back,slot_w_uniform,slot_wear_suit,\
-	slot_wear_mask,slot_head,slot_shoes,slot_gloves,slot_ears,\
-	slot_glasses,slot_belt,slot_s_store,slot_l_store,slot_r_store,slot_wear_id)
-
-	for(var/slot in slots)
-		var/obj/item/I = H.get_item_by_slot(slot)
-		if(I)
-			clone.equip_to_slot_if_possible(I,slot)
```

Comment:
I'm not even sure all this "make the guy have your stuff" code was even functional. In my experience the bodysnatcher popped out with assistant glothes anyway.
Since we're not gibbing anymore I chose to remove all this so there isn't item duplication